harvard-university-of-leeds.csl

```diff
@@ -153,7 +157,7 @@
             <text term="in" text-case="capitalize-first" font-style="italic" suffix=":"/>
             <names variable="editor" delimiter=", " prefix=" " suffix=".">
               <name and="text" delimiter-precedes-last="never" initialize-with=". ">
-                <name-part name="family" text-case="uppercase"/>
+                <name-part name="family" text-case="capitalize-first"/>
```

Comment:
I've just been talking to our referencing gurus about this. The CSL file I was supplied with was edited in Mendeley - and I suspect the 'none' (or however it's described) wasn't appealing to the person updating the style.
So yes, I think this should be removed.